grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.config.ts

```diff
@@ -1,19 +1,17 @@
 import { FormItem, FormItemType } from 'components/GForm/GForm.types';
 
+export type FormData = {
```

Comment:
`FormData` seems to be a very generic wording, makes me think it's used part of the `Form` component, but it's only used for the add responders. Mind if you rename it to something more related to add responders functionality? (e.g. `ManualAlertGroupFormData`)